pkg/xds/topology/outbound.go

```diff
@@ -499,18 +514,11 @@ func NewExternalServiceEndpoint(
 	}, nil
 }
 
-func convertToEnvoy(ds *v1alpha1.DataSource, mesh string, loader datasource.Loader) []byte {
+func loadBytes(ds *v1alpha1.DataSource, mesh string, loader datasource.Loader) ([]byte, error) {
 	if ds == nil {
-		return nil
+		return nil, nil
 	}
-
-	data, err := loader.Load(context.Background(), mesh, ds)
-	if err != nil {
-		core.Log.Error(err, "failed to resolve ingress's public name, skipping dataplane")
-		return nil
-	}
-
-	return data
+	return loader.Load(context.Background(), mesh, ds)
```

Comment:
Should you wire up this context while you are here?

Reply:
Technically yes, but it's quite combersome to get it there.
`outbound.go` uses always static loader which just ignores context.